src/boss/cloudvm/gcp_rest.go

```diff
@@ -210,7 +57,7 @@ const gcpLaunchVmJSON = `{
   },
   "guestAccelerators": [],
   "labels": {},
-  "machineType": "projects/{{.Project}}/zones/{{.Zone}}/machineTypes/e2-small",
+  "machineType": "projects/{{.Project}}/zones/{{.Zone}}/machineTypes/e2-medium",
```

Comment:
and this too?

Reply:
I've added the disk size and machine type fields to the boss's config file. aaaa480dbeef9d9cb4525ba4acb71d845d874421